staging/src/k8s.io/apiserver/pkg/apis/config/types.go

```diff
@@ -74,6 +76,11 @@ type Key struct {
 	Secret string
 }
 
+// String implements Stringer interface in a log safe way.
+func (k Key) String() string {
```

Comment:
was this change required for the external config type as well?

Reply:
Yes. Done.